### ISSUE TITLE
SSL certified has expired

### ISSUE THREAD

**Issue by victor-torres** [luanfonceca/speakerfight#204]
Despite the fact that certbot was configured to automatically renew the certificate via crontab in #165, the website’s certificate is being reported as invalid by browsers because it has expired (invalid date).

I have just tried to run the renew method manually and the following error appears:

> At least one of the required ports is already taken

**Comment by cinnabot** [comment]
Hi, I have some suggestions to help solving this issue!



Here is a list of similar issues:


1. [Enable SSL](https://github.com/luanfonceca/speakerfight/issues/165)

2. [404 after ask to reset e-mail at &quot;reset done&quot;](https://github.com/luanfonceca/speakerfight/issues/67)

3. [Group &quot;My Proposals&quot; by event](https://github.com/luanfonceca/speakerfight/issues/202)

4. [Disallow the acces of the pages by using Decorators.](https://github.com/luanfonceca/speakerfight/issues/17)

5. [Make the deadline more visible](https://github.com/luanfonceca/speakerfight/issues/148)






Please let me know if this was useful:
- [Yes, this was useful](https://www.getcinnamon.io/posts/f57741df-622c-4587-b3dd-0c1d2b3b46f3/rate/?rate=like)
- [No, this wasn't useful](https://www.getcinnamon.io/posts/f57741df-622c-4587-b3dd-0c1d2b3b46f3/rate/?rate=dislike)

Cheers,
Cinnabot.


PS: Liked my suggestions? Install me in your repository! Visit: [getcinnamon.io](https://www.getcinnamon.io).

**Comment by victor-torres** [comment]
Nginx was running on ports 80 and 443 (obviously) but the renew script needs these ports to perform some remote validations. I changed the crontab:

- now we stop nginx before running the renew command and start it again after that
- now the renew command runs at midnight in day 1 of every month (it was running twice a day)

This time I have renewed the cert manually. Let’s see if the issue was completely addressed by the end of the month.
